vendor/github.com/ethereum/go-ethereum/les/backend.go

```diff
@@ -249,6 +250,7 @@ func (s *LightEthereum) Stop() error {
 
 	s.eventMux.Stop()
 
+	time.Sleep(time.Millisecond * 200)
```

Comment:
is it a leftover or still required?

Reply:
i guess leftover, i don't see this line upstream

Reply:
yep, it is a leftover, let me take a look...

Reply:
fixed it :)